ci/steps.yml

```diff
@@ -285,7 +290,13 @@ steps:
         cd $(Build.BinariesDirectory)\tests
         7z x $(Build.SourcesDirectory)\ci\helloworld.7z
         cd ..
-        qmake $(Build.BinariesDirectory)\tests\helloworld
+        
+        # Add QMAKE_CXXFLAGS for MSVC 2022
```

Comment:
As a separate PR should we transition from qmake to cmake?  gpsbabel maintenance certainly got much easier when we did so.

Reply:
It's a possibility, but I usually consider it like this:
- CMake is more generalist, and is the one I use by default because I'm used to it and it plays nice with additional libraries
- QMake is the faster and easier way to get it to work

Since it's just about installing Qt to test, I would say QMake is a better choice to not get tangled in CMake when we just want to test that our installs build

Reply:
Most of these examples came from Qt, and they have converted them to cmake (adding CMakeLists.txt and deleting the *.pro files).  But I would suggest this is outside the scope of your PR anyway.